### PR TITLE
Install python scripts using entry_points

### PR DESCRIPTION
**Pull request by jeffa5**
This ensures that they will be runnable with just the filename, not requiring something like `python3 $(which read_ledger.py)`.

c.f. https://docs.python.org/3/distutils/setupscript.html#installing-scripts

### CODE REVIEW

**Comment by jeffa5** [comment]
Note, installing the python package with `pip install -e ./python` will do other things which are different from `pip install ./python` which this is fixing.

**Review by jumaffre — APPROVED** [review]
Thank you!

**Comment by jeffa5** [comment]
After chatting with @achamayou I'm going to update it to use entry_points

File: python/ccf/split_ledger.py [hunk 1]
```diff
@@ -168,12 +168,15 @@ def run(args_):
...
     LOG.remove()
     LOG.add(
         sys.stdout,
         format="<level>{message}</level>",
     )
 
     run(sys.argv[1:])
+
+
+if __name__ == "__main__":
```
Comment (achamayou) [inline python/ccf/split_ledger.py:181]: This is unnecessary when using entry_points.

File: python/ccf/read_ledger.py [hunk 2]
```diff
@@ -244,3 +243,7 @@ def run(
         args.digests_only,
     ):
         sys.exit(1)
+
+
+if __name__ == "__main__":
```
Comment (achamayou) [inline python/ccf/read_ledger.py:248]: This is unnecessary when using entry_points.

**Review by MahatiC — APPROVED** [review]

**Review by achamayou — APPROVED** [review]

**Pull request merged**